src/main/java/org/commcare/formplayer/application/GlobalDefaultExceptionHandler.java

```diff
@@ -51,9 +51,8 @@ public class GlobalDefaultExceptionHandler {
             NoLocalizedTextException.class})
     @ResponseBody
     public ExceptionResponseBean handleApplicationError(HttpServletRequest request, Exception exception) {
-        log.error("Request: " + request.getRequestURL() + " raised " + exception);
+        log.info("Application error", exception);
         datadog.incrementErrorCounter(Constants.DATADOG_ERRORS_APP_CONFIG, request);
-        FormplayerSentry.captureException(exception, SentryLevel.INFO);
```

Comment:
This was creating an event before but now will not. I don't think there is value in having application error events in Sentry.

The log event is still captured in Sentry but as breadcrumbs

Reply:
Breadcrumbs are only available in sentry when there is an error event right ?